### ISSUE TITLE
Unified wheel package is missing dependency on 'ipaddress' for Python 2.7

### ISSUE THREAD

**Issue by edmorley** [elastic/elasticsearch-dsl-py#803]
Hi!

After installing `elasticsearch-dsl` under Python 2.7 using latest pip, the first time I import it, I get:
`ImportError: No module named ipaddress`

I believe this is because the dependency on `ipaddress` is defined like so:
https://github.com/elastic/elasticsearch-dsl-py/blob/6af6b8cd26ef13bf42b69a92f0429141e2d9bd0d/setup.py#L20-L22

...and for wheels, setup.py is not evaluated on install, only during the initial creation of the wheel - so if the wheel is built under Python 3, then the dependency is missing from the metadata created under `elasticsearch_dsl-6.0.0.dist-info` even when the wheel is actually installed/used under Python 2.7.

As such, either:
* `elasticsearch-dsl` needs to be distributed as two separate "[pure Python wheels](https://packaging.python.org/tutorials/distributing-packages/#pure-python-wheels)" (rather than as a "[unified wheel](https://packaging.python.org/tutorials/distributing-packages/#universal-wheels)") - one built under Python 2, and the other built under Python 3
* the `install_requires` entry needs to be defined using the special python version specifier syntax, like so:
```python
install_requires =[
    # ...
    'ipaddress; python_version < "3.3"',
]
```
([example usage in another project for reference](https://github.com/django/channels/blob/db1b3ba951ef2c3cff6e91ed94d4115d479c0d4f/testproject/setup.py#L10))

The second option seems preferable since it avoids making the build/release process more complex :-)

**Comment by HonzaKral** [comment]
Thank you, @edmorley, for a quick and detailed report! Fixed package released as `6.0.1`.

**Comment by edmorley** [comment]
You're welcome - many thanks for the quick fix! :-)
